projects/observability/src/shared/components/cartesian/d3/cartesian-no-data-message.ts

```diff
@@ -8,7 +8,7 @@ export class CartesianNoDataMessage {
 
   public constructor(
     private readonly hostElement: Element,
-    private readonly series: Series<{}>[],
+    private readonly series: Series<unknown>[],
```

Comment:
why this change?

Reply:
reverted

Reply:
scratch that. This is the error we get from the new ts

```
Argument of type 'Series<TData>[]' is not assignable to parameter of type 'Series<{}>[]'.
  Type 'Series<TData>' is not assignable to type 'Series<{}>'.
    Type 'TData' is not assignable to type '{}'.ts(2345)
    
```

Reply:
adding a bound on the definition, `Series<TInterval>` would resolve that right?

Reply:
umm.. may be, but unknown makes sense to me here. No data doesn't need to know the generic type of series data. I can define TInterval, but i think i will also have to default to unknown or always pass a type from the caller side.

Reply:
In the context of the larger review, didn't look where we were. Agree, no data doesn't care (i.e. doesn't make any assumptions that it's an object) and as long as it works with unknown it's fine.